index.html

```diff
@@ -0,0 +1,106 @@
+<html>
+    <head>
```

Comment:
I think it would be better to use <!DOCTYPE html> befor <html> tag.

Reply:
- [ ] Well done on using inline comments and giving a polite suggestion. 👍 
- Kindly give a reason `why` it is `better to use < !DOCTYPE html > befor < html > tag.`